en/starter/generator/command_line_tools_for_linux.md

```diff
@@ -30,6 +31,29 @@ generator and its prerequisites are installed correctly.
 > * [IBM Cloud Developer Tools CLI](https://console.ng.bluemix.net/docs/cloudnative/tutorial_web.html#tutorial/) via `bx dev create`  
 > (only supports scaffolded applications).
 
+## Installation
+
+<span class="arrow">&#8227;</span> Install [Prerequisites](/{{ page.lang }}/starter/settingup.html).
+
+<span class="arrow">&#8227;</span> Complete the following steps to install Swift Server Generator:
+
+1.  Open a Terminal window.
+
+2.  You will need the Yeoman command line utility [yo](https://github.com/yeoman/yo) installed in your global Node.js module directory:
+
+        $ npm install -g yo
+
+3.  To install [Swift Server Generator](https://github.com/IBM-Swift/generator-swiftserver), run the following command:
+
+        $ npm install -g generator-swiftserver
+
+> ![info] Note: Swift Server Generator is supported only on the following operating systems:
+>
+> -   Ubuntu Linux 14.04 LTS
+> -   Ubuntu Linux 16.04 LTS
+
```

Comment:
If we change the documentation as suggested then surely macOS has to be added to this bulleted list of operating systems too?

Reply:
The `yo swiftserver` generator seems to work fine on Mac, but I don't know how we decide what is 'supported' - @tunniclm any thoughts?

Reply:
We support it on anything Node.js runs on, so definitely macOS too. I think I would just remove the info box to be honest.